### ISSUE TITLE
Browser extensions are the more secure / recommended way of using the tool

### ISSUE THREAD

**Issue by MiguelRipoll23** [lesspass/lesspass#63]
I quite like the idea of this but due to this repository being so popular, I think that recommending people to compute their password on a website (not hosted by them) is a really bad security practice. If the website gets hacked or the owner of the site decides to log everything they can receive the password computed to login.

Again, I really like the idea and sounds exciting.

**Comment by guillaumevincent** [comment]
**TL;DR** use the web extension

We should encourage people use the web extension for sure.
This is why there is a lot of differents ways to use LessPass: 

- web extension
- self hosted
- MyCozy app
- site
- saved on local
- npm
- command line
- github page

But I do not agree with the sentence "really bad security pratice". If LessPass being so popular, there will be people to alert in case of problems. Never trust us ! In offline mode check that no request are made client side. Look at the source code ([we will remove obfuscated code](https://github.com/lesspass/lesspass/issues/61))

We don't want people to trust us. We want users understand and regain control over their password managers.

Passwords are built client side. So if we want to log, it will be client-side.
On the server we can log which ip reach our server at what time.

And self hosted came with other security problems. People have hosted wordpress sites were they more secure than people hosting their sites on wordpress.com? I'm sure you got the answer. I no a lot of small company hacked because their sites wasn't updated. 

Glad you like the idea

Where do we recomend people to use the site ?

**Comment by guillaumevincent** [comment]
We should rename this: **Recommend people to use the web extension**

**Comment by guillaumevincent** [comment]
We should display a warning on the frontpage and encourage people to use the web extension

**Comment by MiguelRipoll23** [comment]
Problem is that needing to check the source every time you need to compute your password is quite inconvenient and some mobile browsers don't allow to do this, I also know that most of us here on GitHub know what we are doing and we're cautious!

Here:
> Is it available on my OS/device?
> Yes, as soon as you have access to a browser it’s available to you. But we went behind that and added:
> a Chrome extension ;
> a Firefox extension ;
> a Cozy application (if you want to self-host it) ;
> **and the official site.**

Thanks for the great responses about this.

**Comment by guillaumevincent** [comment]
@MiguelRipoll23 thanks we will put a caution on the blog post too
